posts/2021-01-06-ocp-log-forwarding.adoc

```diff
@@ -13,22 +13,22 @@ additional_authors:
   image: https://avatars0.githubusercontent.com/Channyboy
 ---
 = Forwarding Open Liberty logs in OpenShift to Splunk by using the Log Forwarding API
-Halim Lee <https://github.com/leemelim>
+Halim Lee <https://github.com/melissa-lee>
 
 Collecting logs in a single location helps you monitor all of your applications efficiently, which is especially important in highly distributed and dynamic container orchestration systems. Red Hat OpenShift Container Platform (OpenShift) provides a built-in log aggregation solution that uses Elasticsearch, Fluentd, and Kibana, also known as the EFK stack. In addition to the EFK stack, you can use the https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-external.html[Log Forwarding API] to send logs from an OpenShift cluster to other log analysis solutions. The Log Forwarding API is available starting with OpenShift 4.6, and in OpenShift 4.3 to 4.5 as a Technology Preview feature.
 
 https://www.splunk.com/[Splunk] is one of the most popular log analysis solutions. If you want to start using Splunk as your logging solution for Open Liberty on OpenShift, you must first integrate OpenShift with Splunk. The Log Forwarding API simplifies OpenShift cluster integration with Splunk. You can configure custom pipelines to send logs to endpoints that are either inside or outside of your OpenShift cluster.
 
-To send logs from OpenShift to Splunk, the Log Forwarding API must first send them to a Fluentd server. This blog post walks you through configuring the Log Forwarding API on your OpenShift cluster to forward your logs to a Fluentd and Splunk deployment that is external to your cluster.  We also cover the configuration and deployment of the Fluentd and Splunk server on an external machine. The Fluentd server redirects the forwarded logs to Splunk by using Splunk's HTTP Event Collector (HEC) API. In the final sections, you integrate your Open Liberty logs with the Fluentd forwarder and setup Splunk dashboards that are provided by the Open Liberty team.
+To send logs from OpenShift to Splunk, the Log Forwarding API must first send them to a Fluentd server. This blog post walks you through configuring the Log Forwarding API on your OpenShift cluster to forward your logs to a Fluentd and Splunk deployment that is external to your cluster.  We also cover the configuration and deployment of the Fluentd and Splunk server on an external machine. The Fluentd server redirects the forwarded logs to Splunk by using Splunk's HTTP Event Collector (HEC) API. In the final sections, if JSON logging is enabled for Open Liberty, you will integrate your Open Liberty JSON  logs with the Fluentd forwarder and setup Splunk dashboards that are provided by the Open Liberty team.
```

Comment:
In the final sections, if JSON logging is enabled for Open Liberty, you will integrate your Open Liberty JSON  logs with the Fluentd forwarder and setup Splunk dashboards that are provided by the Open Liberty team.
->
In the final sections, you integrate your Open Liberty logs with the Fluentd forwarder and access the logs in Splunk.  If JSON logging is enabled for Open Liberty you can also work with your logs using a dashboard provided by the Open Liberty team.